ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/db/ManifestAssociationDao.java

```diff
@@ -54,4 +54,14 @@
             "WHERE assc.manifest_key = :manifestID")
     LiveData<List<Article>> getArticlesByManifestID(String manifestID);
 
+
+    //region Testable calls (No Live Data)
```

Comment:
there are some annotations that signify to AS restricted scope so that these methods aren't used in incorrect places.

Reply:
need to look up which ones are appropriate :)